src/main/java/httpclient/controllers/DashboardHandler.java

```diff
@@ -0,0 +1,36 @@
+package httpclient.controllers;
+
+import httpclient.client.BaseRequest;
+import httpclient.client.HttpClientBase;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpResponse;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class DashboardHandler extends BaseRequest {
```

Comment:
could you please clarify this inheritance? is DashboardHandler a BaseRequest?

Reply:
DashboardHandler, WidgetHandler extend BaseRequest.
Maybe there are not very good names of classes, e.g. DashboardRequest, WidgetRequset are better

Reply:
yes it will be more clear if rename

Reply:
done

Reply:
now it's "DashboardController extends BaseRequest", it's almost the same as previously "DashboardHandler extends BaseRequestBaseRequest"